builder/builder-next/worker/worker.go

```diff
@@ -7,9 +7,11 @@ import (
 	nethttp "net/http"
 	"time"
 
+	containerdsnapshot "github.com/moby/buildkit/snapshot/containerd"
+	"github.com/moby/buildkit/util/leaseutil"
+
```

Comment:
nit: remove newline

Reply:
Is there any tool that would automatically format imports properly?

Reply:
I don't think there is any rule in goimports or depguard to handle this case unfortunately.

Reply:
🙈